pkg/kubectl/rolling_updater.go

```diff
@@ -208,9 +208,11 @@ func (r *RollingUpdater) Update(config *RollingUpdaterConfig) error {
 	if err != nil {
 		return err
 	}
-	// Further validation.
-	if maxUnavailable == 0 && maxSurge == 0 {
```

Comment:
Yeah, you're right. I'm going to have to force update to fix my committer email address.